### PR TITLE
#51 Improve seo support

### PR DESCRIPTION
**Pull request by hlohaus**

### CODE REVIEW

File: src/Pwa/PageLoader/ProductPageLoader.php [hunk 1]
```diff
@@ -88,10 +84,11 @@ public function load(PageLoaderContext $pageLoaderContext): ProductPageResult
             new EqualsFilter('active', 1)
         );
 
+        $criteria->addAssociation('seoUrls');
```
Comment (elkmod) [inline src/Pwa/PageLoader/ProductPageLoader.php:87]: I think we don't need to specify that association, as it is loaded whenever we include the `sw-include-seo-urls` header for our request. We used to have a small bug regarding that resolving for sales channel entities which should have been fixed in 6.3.* and will be available very soon.

**Comment by elkmod** [comment]
Merged with [ebdd77a](https://github.com/elkmod/SwagShopwarePwa/commit/ebdd77ae4de3be94511bbbd60be9bea2144ea89f).

**Comment by elkmod** [comment]
Thanks for your PR, I've added some tests and merged it!

**Comment by elkmod** [comment]
We still need a test here, and could you please motivate, why we need the entire category within the response?

**Comment by elkmod** [comment]
After discussion with Josip (via Slack Channel) this PR was reopened - reason: Category contains meta information which is relevant for displaying SEO information. A corresponding feature request in shopware-pwa will be placed.

**Comment by develth** [comment]
Anything todo here that i could support to get it done?

**Pull request merged**